tests/build/chains.go

```diff
@@ -66,35 +68,58 @@ var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", func() {
 			Namespace:  constants.TEKTON_CHAINS_NS,
 		}
 
+		// the default policy source
+		rev := "main"
+		policySource := ecp.GitPolicySource{
+			Repository: "https://github.com/hacbs-contract/ec-policies.git",
+			Revision:   &rev,
+		}
+
+		// if there is a ConfigMap e2e-tests/ec-config with keys `revision` and
+		// `repository` values from those will replace the default policy source
+		// this gives us a way to set the tests to use a different policy if we
+		// break the tests in the default policy source
+		if config, err := framework.CommonController.K8sClient.KubeInterface().CoreV1().ConfigMaps("e2e-tests").Get(context.TODO(), "ec-config", v1.GetOptions{}); err != nil {
```

Comment:
Oh, we shouldn't do this in a container node: https://onsi.github.io/ginkgo/#mental-model-how-ginkgo-traverses-the-spec-hierarchy. 

Maybe toss it under `BeforeAll`?

Reply:
Yeah, makes sense